packages/sdk/src/types/Orders.ts

```diff
@@ -1,6 +1,485 @@
 import { CustomerRequest } from './Customers'
+import { FilterConditionsNumber, FilterConditionsString, FilterJunction } from './Exports'
 
-export interface OrdersItem {
```

Comment:
Interface vs Type - please look at origin comment about the `DistributionsPublicationsCreateParams`